### PR TITLE
fix: bulk update event type dialog logic

### PR DESCRIPTION
**Pull request by anikdhabal**
## What does this PR do?

Issue:-

https://github.com/user-attachments/assets/f6f55765-788e-4595-bc00-41bd268683ab

https://github.com/user-attachments/assets/4098a14b-575e-4ccb-b45e-e43a67d1c04b

## Mandatory Tasks (DO NOT REMOVE)

- [x] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [x] N/A-I have added a Docs issue [here](https://github.com/calcom/docs/issues/new) if this PR makes changes that would require a [documentation change](https://docs.cal.com). If N/A, write N/A here and check the checkbox.
- [x] N/A-I confirm automated tests are in place that prove my fix is effective or that my feature works.

## How should this be tested?

- Are there environment variables that should be set?
- What are the minimal test data to have?
- What is expected (happy path) to have (input and output)?
- Any other important info that could help to test that PR

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my changes generate no new warnings

### CODE REVIEW

File: apps/web/components/apps/AppList.tsx [hunk 1]
```diff
@@ -98,7 +99,6 @@ export const AppList = ({ data, handleDisconnect, variant, listClassName }: AppL
                             updateDefaultAppMutation.mutate({
                               appSlug,
                             });
-                            setBulkUpdateModal(true);
```
Comment (anikdhabal) [inline apps/web/components/apps/AppList.tsx:101]: This is not the right place to add this; it should only be true if the mutation is successful

**Comment by Amit91848** [comment]
@anikdhabal can you confirm this occurs on main too? I checked it on app.cal.com and did not face any issue.


https://github.com/user-attachments/assets/955676b1-8443-457e-93a5-d82a3a438199

**Comment by anikdhabal** [comment]
> @anikdhabal can you confirm this occurs on main too? I checked it on app.cal.com and did not face any issue.

Nah, tit's fine. This PR is not about the error with setting the static link as default. I just moved the dialog open action to onSuccess

**Comment by Amit91848** [comment]
Yeah figured

**Review by Amit91848 — APPROVED** [review]
LGTM!

**Pull request merged**